vnext/ReactUWP/Modules/Animated/AnimationDriver.cpp

```diff
@@ -36,42 +36,45 @@ AnimationDriver::~AnimationDriver() {
 void AnimationDriver::StartAnimation() {
   const auto [animation, scopedBatch] = MakeAnimation(m_config);
 
-  const auto animatedValue = GetAnimatedValue();
+  if (auto const animatedValue = GetAnimatedValue()) {
+    auto const previousValue = animatedValue->Value();
+    auto const rawValue = animatedValue->RawValue();
+    auto const offsetValue = animatedValue->Offset();
 
-  if (animatedValue) {
     animatedValue->PropertySet().StartAnimation(
         ValueAnimatedNode::s_offsetName, animation);
     animatedValue->AddActiveAnimation(m_id);
   }
   scopedBatch.End();
 
-  m_scopedBatchCompletedToken = scopedBatch.Completed(
-      [endCallback = m_endCallback, animatedValue, id = m_id](
-          auto sender, auto) {
-        if (endCallback) {
-          endCallback(std::vector<folly::dynamic>{
+  m_scopedBatchCompletedToken =
+      scopedBatch.Completed([this](auto sender, auto) {
```

Comment:
This event token is being saved, but I don't see anything that's protecting the [this] capture. The event could happen after the lifetime of this object, correct?